assets/javascripts/discourse/components/webinar-register.js.es6

```diff
@@ -82,16 +82,16 @@ export default Component.extend({
     return false;
   },
 
-  @discourseComputed("webinar.starts_at", "webinar.attendees")
-  canUnregister(starts_at, attendees) {
+  @discourseComputed("webinar.starts_at")
+  canUnregister(starts_at) {
```

Comment:
This should declare dependency on `isAttendee` and `registered`. The same applies to `userCanRegister()` below.